digits/model/tasks/caffe_train.py

```diff
@@ -26,6 +26,7 @@
 # Must import after importing digit.config
 import caffe
 import caffe_pb2
+import platform
```

Comment:
Can you move this up just before `import scipy`?

I'd like to stick with this style: https://github.com/NVIDIA/DIGITS/pull/501